StartHere.ipynb

```diff
@@ -13,45 +13,45 @@
     "[The Jupyter documentation](http://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Running%20Code.html) covers interacting with code cells, markdown, and notebooks. The tutorials below show the features of the BeakerX extension.\n",
     "\n",
     "### Languages\n",
-    "[Groovy](doc/groovy/Groovy.ipynb), [Java](doc/java/Java.ipynb), [Scala](doc/scala/Scala.ipynb), [Clojure](doc/clojure/Clojure.ipynb), [SQL](doc/sql/SQL.ipynb), [Kotlin](doc/kotlin/Kotlin.ipynb).\n",
+    "[Groovy](groovy/main.ipynb), [Java](java/main.ipynb), [Scala](scala/main.ipynb), [Clojure](clojure/main.ipynb), [SQL](sql/main.ipynb), [Kotlin](kotlin/main.ipynb).\n",
```

Comment:
uh since "doc" was removed, none of these links work...

Reply:
I had this file in doc file, and then moved it to the top and forget about bring back doc/